### ISSUE TITLE
TL;DR Please update the README with the actual system requirements to run this.

### ISSUE THREAD

**Issue by efitzg** [carykh/alignedCelebFaces#13]
TL;DR Please update the README with the actual system requirements to run this.

So, my son saw your YouTube video and asked me to download this so he could play with it. I am trying to get him more interested in programming so I thought, "Sure, why not?" 

I followed your install instructions and Windows puked all over itself. I didn't want to mess up my gaming machine so I stood up an Ubuntu 18.04 virtual box (I am very lucky I happened to have VirtualBox installed and an Ubuntu iso recently downloaded) and installed git, cloned the repo and tried to install there. It said I didn't have pip so I installed pip and tried again. It said I don't have cmake. So I installed cmake and tried again. It said "Failed building wheel for dlib" waited an hour then told me it was out of memory. I doubled the amount of ram assigned to the virtual machine and tried again. Finally, it works! 

To save the next guy so many hours of frustration, please give a little more detail on what is required to actually run this.

P.S. Thanks for getting my son interested in machine learning. I think we will have fun with it together.
